dev/prod_builders.json

```diff
@@ -72,12 +72,6 @@
          "task_name": "linux_gradle_plugin_fat_apk_test",
          "flaky": false
       },
-      {
-         "name": "Linux gradle_plugin_light_apk_test",
-         "repo": "flutter",
-         "task_name": "linux_gradle_plugin_light_apk_test",
-         "flaky": false
-      },
```

Comment:
Is this intended to remove?

Reply:
Yes, that one is still failing consistently.